### DISCUSSION TITLE
Reddit group inactivity

### DISCUSSION THREAD

**Discussion by lollisoft** [open-watcom/open-watcom-v2#1270] [General]
Hi,

I just like to ask where it is better to chat about the old Sybase Power++ product. The Reddit https://www.reddit.com/r/OpenWatcom/ group seems the wrong place.

Are there any ex Power++ users still available to exchange knowledge?

Thanks

Lothar

**Comment by jmalak** [comment]
Hi Lothar,

I don't know what Sybase Power++ products are.
Here we work on Open Watcom compilers (original WATCOM, next Sybase).
I am not sure if we can help anyway.
May be some community for Sybase Power++ product exists.
We have our own [Discord server channels](https://discord.gg/39w5wZM89p) that you can try to ask there.

Regards

Jiri

**Reply by lollisoft** [reply]
Thank you. First I tried to enter the discord group via the browser but my OS (Mac) seems too old. I am in the channel now from my mobile phone.
